src/app/listener.ts

```diff
@@ -24,6 +25,12 @@ export class Listener {
     this.container.clientService.on('ready', () => {
       this.container.loggerService.info(`Loaded ${this.container.jobService.size()} jobs...`);
       this.container.loggerService.info('Lion is now running!');
+
+      //Cache messages in #code_of_conduct to enable the onMessageReact Handler for them
+      const cocChannel = this.container.guildService
+        .get()
+        .channels.find((c) => c.name == 'code_of_conduct') as TextChannel;
+      cocChannel.fetchMessages({ limit: 10 });
```

Comment:
this is necessary?

Reply:
Yes, `on('messageReactionAdd'...)` only works on cached messages. Message are only cached if fetched, or sent after the bot starts tunning

Reply:
gotcha.